### ISSUE TITLE
Implement plugin settings in admin console

### ISSUE THREAD

**Issue by mickmister** [mattermost-community/mattermost-plugin-welcomebot#108]
The configuration for the plugin is currently handled directly in the server's `config.json`, which makes it difficult to manage since you can't use the MM UI to do so.

The task here is to declare these config settings in the plugin's `plugin.json` file, and implement any complex settings as a custom setting in the admin console.

**Comment by Kshitij-Katiyar** [comment]
@mickmister I have a clarification about this. 
If this ticket is urgent just for the sake of this plugin working with the cloud servers. We can create a long text field, put that json into that field in the plugin configuration, and parse that from there. This would be a quick solution to make this work with cloud servers.
The second solution would be to create custom components in the web app to get all these settings from the plugin configurations. This would take longer as we are creating customs components and this would not work in mobile apps as well.

**Comment by mickmister** [comment]
@Kshitij-Katiyar Note that there is progress on this, using the simpler approach you mentioned https://github.com/mattermost/mattermost-plugin-welcomebot/pull/92. However, I'm still thinking we should implement as custom components.

**Comment by Kshitij-Katiyar** [comment]
@mickmister I have made some components and also registered the same in the system console settings. Can you please take a look and give your suggestions for the same. 

![Screenshot from 2023-07-14 17-09-17](https://github.com/mattermost/mattermost-plugin-welcomebot/assets/90389917/cca061ba-4926-44f1-abf6-27845879821f)
![Screenshot from 2023-07-14 17-09-26](https://github.com/mattermost/mattermost-plugin-welcomebot/assets/90389917/0c3a9f0d-1b02-4d4c-bcf0-c3fae4f6ed93)

**Comment by mickmister** [comment]
@raghavaggarwal2308 Is this PR still being worked on? https://github.com/Brightscout/mattermost-plugin-welcomebot/pull/15

**Comment by mickmister** [comment]
Asking so we can determine whether to move forward with that PR or https://github.com/mattermost/mattermost-plugin-welcomebot/pull/92

**Comment by raghavaggarwal2308** [comment]
@mickmister Yes, we are working on https://github.com/Brightscout/mattermost-plugin-welcomebot/pull/15. We will create a PR in mattermost org ASAP.

**Comment by Kshitij-Katiyar** [comment]
> Asking so we can determine whether to move forward with that PR or #92

@mickmister This is taking longer than expected, we can move with the other PR https://github.com/mattermost/mattermost-plugin-welcomebot/pull/92 in the meantime if you want.

**Comment by mickmister** [comment]
@Kshitij-Katiyar Sounds good to me. I'm thinking the changes will be backwards compatible for both PRs. Are you able to drive the other PR forward? Thanks for pointing this out @Kshitij-Katiyar

**Comment by c-seeger** [comment]
@Kshitij-Katiyar any plans for this feature?
